elastio-stream-oracle-database-backup/.env

```diff
@@ -0,0 +1,6 @@
+########################
+# DATABASE CREDENTIALS #
+########################
+USERNAME="admin"
+PASSWORD="h30sb06m8uUgtwMwAK1"
```

Comment:
Probably should `.gitignore` this file, even if these credentials are no longer valid.